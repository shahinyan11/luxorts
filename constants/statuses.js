export const PASSWORD_UPDATE_SUCCESS_STATUS = 'success';

export const ACCOUNT_STATUS = {
  DEACTIVATED_BY_USER: 'deactivated_by_user',
};

export const ACTIVE = 'active';
