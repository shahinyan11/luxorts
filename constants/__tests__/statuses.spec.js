import * as constants from '../statuses';

it('statuses constants match snapshot', () => {
  expect(constants).toMatchSnapshot();
});
