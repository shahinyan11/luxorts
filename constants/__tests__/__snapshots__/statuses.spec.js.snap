// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`statuses constants match snapshot 1`] = `
Object {
  "ACCOUNT_STATUS": Object {
    "DEACTIVATED_BY_USER": "deactivated_by_user",
  },
  "ACTIVE": "active",
  "PASSWORD_UPDATE_SUCCESS_STATUS": "success",
}
`;
